operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -208,8 +208,22 @@ type DseWorkloads struct {
 	SearchEnabled    bool `json:"searchEnabled,omitempty"`
 }
 
+// StorageConfig defines additional storage configurations
+type AdditionalVolumes struct {
+	// Mount path into cassandra container
+	MountPath string `json:"mountPath"`
+	// Name of the pvc
+	// +kubebuilder:validation:Pattern=[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*
+	Name string `json:"name"`
+	// Persistent volume claim spec
+	PVCSpec *corev1.PersistentVolumeClaimSpec `json:"pvcSpec"`
```

Comment:
not sure if the codegen rules would prefer this to be `PvcSpec`... I'm going to find them

Reply:
I expect it to accept it cause that's what is used in the API, for instance see https://godoc.org/k8s.io/kubernetes/pkg/api/persistentvolumeclaim#DropDisabledFields, they don't write it oldPvcSpec

Reply:
nothing to see here, I think older versions of the code generation tools had more requirements